source/Plugins/ExpressionParser/Swift/SwiftASTManipulator.cpp

```diff
@@ -1155,6 +1155,8 @@ bool SwiftASTManipulator::AddExternalVariables(
         if (!m_wrapper_decl)
           continue;
 
+        // We need to mutate the $__lldb_wrapped_expr_%d member of self later.
```

Comment:
Can you elaborate this?

Reply:
The code in https://github.com/apple/swift-lldb/blob/5db61e041f7b3d5e84bce27966e32b351c648946/source/Plugins/ExpressionParser/Swift/SwiftASTManipulator.cpp#L196 doesn't compile without making `$__lldb_injected_self` mutable.